android-components/components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -54,53 +67,58 @@ data class ContextMenuCandidate(
             context: Context,
             tabsUseCases: TabsUseCases,
             contextMenuUseCases: ContextMenuUseCases,
-            snackBarParentView: View,
+            snackbarParentView: View,
             snackbarDelegate: SnackbarDelegate = DefaultSnackbarDelegate(),
         ): List<ContextMenuCandidate> = listOf(
             createOpenInNewTabCandidate(
                 context,
                 tabsUseCases,
-                snackBarParentView,
+                snackbarParentView,
                 snackbarDelegate,
             ),
             createOpenInPrivateTabCandidate(
                 context,
                 tabsUseCases,
-                snackBarParentView,
+                snackbarParentView,
```

Comment:
nit: Renamings like this or changes that are not needed for the issue at hand are usually avoided since they grow the size of the PR and make the review process harder.

Reply:
Appreciate that, I'm mindful of this 🙂 I felt for this case that there was a small enough number of changes and the comparison would be straightforward. Thinking about it now, a separate commit would have been useful. For these kinds of changes, is it recommended creating a separate issue?

Reply:
Just FYI, since changing to use the existing feature it does not include this renaming